speech/mml2mml

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env -S node -r esm
+
+/*************************************************************************
+ *
+ *  speech/mml2mml
+ *
+ *  Uses MathJax v3 to convert a MathML string to a MathML string with alttext.
+ *
+ * ----------------------------------------------------------------------
+ *
+ *  Copyright (c) 2019 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+//
+//  Get the command-line arguments
+//
+var argv = require('yargs')
+    .demand(0).strict()
+    .usage('$0 [options] "math" > file.html')
+    .options({
+        inline: {
+            boolean: true,
+            describe: "process as inline math"
+        },
+        sre: {
+            array: true,
+            nargs: 2,
+            describe: 'SRE flags as key value pairs, e.g., "--sre locale de --sre domain clearspeak" generates speech in German with clearspeak rules'
+        },
+        speech: {
+            default: 'shallow',
+            describe: 'level of speech: deep, shallow, none'
+        },
+        dist: {
+            boolean: true,
+            default: false,
+            describe: 'true to use webpacked version, false to use mathjax3 source files'
+        }
+    })
+    .argv;
+
+const action = require('./action.js');
+
+//
+// Add a render action to move the computed speech into the alttext attribute.
+//
+function moveSpeech(math) {
+  let alttext = '';
+  math.root.walkTree(node => {
+    const attributes = node.attributes.getAllAttributes();
+    console.log(attributes);
```

Comment:
Is this `console.log()` for debugging, and should be removed?

Reply:
Removed.